dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/DSpaceRestRepository.java

```diff
@@ -407,4 +408,21 @@ protected void patch(Context context, HttpServletRequest request, String apiCate
         throw new RepositoryMethodNotImplementedException("No implementation found; Method not allowed!", "");
     }
 
+    public T put(HttpServletRequest request, String apiCategory, String model, ID id, JsonNode jsonNode) {
```

Comment:
Missing JavaDoc for new public method.

Reply:
The JavaDoc has been written now

Reply:
> Method to support updating a DSpace instance.

*DSpace instance*? Maybe *DSpaceObject instance*?

Reply:
It's actually not limited to a DSpaceObject.
I've adjusted it to be similar to the patch comment:
`Apply an update to the REST object via JSON PUT`